hotels-backend/Hotels/Extensions/IdentityExtension.cs

```diff
@@ -12,10 +12,11 @@ public static class IdentityExtension
     {
         public static int GetAuthorizedUserId(this ClaimsIdentity identity)
         {
-            return Convert.ToInt32(identity.Claims
-                .Where(x => x.Type == ClaimTypes.NameIdentifier)
-                .FirstOrDefault()
-                .Value);
+            return Convert.ToInt32(
+                identity.Claims
+                    .Where(x => x.Type == ClaimTypes.NameIdentifier)
+                    .FirstOrDefault()
+                    .Value);
```

Comment:
```suggestion
            return Convert.ToInt32(
                identity.Claims
                    .Where(x => x.Type == ClaimTypes.NameIdentifier)
                    .FirstOrDefault()
                    .Value
            );
```

In such way you should clearly see where corresponding method call ends;